__tests__/app.test.js

```diff
@@ -40,4 +40,24 @@ describe('/*', () => {
             expect(msg).toBe("Path not found.");
         })
     });
+});
+
+describe('/api', () => {
+    test('GET: 200 responds with an object describing all the available endpoints on the api', () => {
+        return request(app)
+        .get('/api')
+        .expect(200)
+        .then(({ body: { apiDocs } }) => {
+            for (const endpoint in apiDocs) {
+                expect(apiDocs[endpoint]).toHaveProperty('description');
+                expect(typeof apiDocs[endpoint].description).toBe('string');
+                expect(apiDocs[endpoint]).toHaveProperty('queries');
+                expect(Array.isArray(apiDocs[endpoint].queries)).toBe(true);
+                expect(apiDocs[endpoint]).toHaveProperty('exampleBody');
+                expect(typeof apiDocs[endpoint].exampleBody).toBe('object');
+                expect(apiDocs[endpoint]).toHaveProperty('exampleResponse');
+                expect(typeof apiDocs[endpoint].exampleResponse).toBe('object');
```

Comment:
nice extemsive test, though you could require in the endpoints.json to the test file and compare what the /api endpoint returns to that.